docs/upgrade/upgrade-opensource/upgrade-guide-from-5.4-to-6.0/metric-update-5.4-to-6.0.rst

```diff
@@ -0,0 +1,16 @@
+ScyllaDB Metric Update - Scylla 5.4 to 5.5
+============================================
+
+.. toctree::
+   :maxdepth: 2
+   :hidden:
+
+Scylla 5.5 Dashboards are available as part of the latest |mon_root|.
```

Comment:
6.0

Reply:
So how about these consistency updates? "5.5" is still mentioned in a bunch of places in this PR :(

We should be probably using a variable like `|NEW_VERSION|` to avoid this problem